### PR TITLE
[coinmate] new features

### PR DESCRIPTION
**Pull request by npomfret**
* higher resolution nonce
 * added fetchMyTrades and some missing api endpoints
 * added fetchTransactions

### CODE REVIEW

File: js/coinmate.js [hunk 1]
```diff
@@ -281,6 +473,10 @@ module.exports = class coinmate extends Exchange {
         return await this.privatePostCancelOrder ({ 'orderId': id });
     }
 
+    nonce () {
```
Comment (kroitor) [inline js/coinmate.js]: Is it required? They won't accept more than 100 requests per minute, that could be roughly compared to 1 per minute... What do you think?

Comment (npomfret) [reply js/coinmate.js]: It allows for faster bursts or requests, and in the case where you are hitting the exchange in multiple threads (without in-build throttling) it reduces the chance of nonce-reuse.  In general I would always favour the highest resolution nonce that the exchange supports.

**Comment by npomfret** [comment]
Regarding your comment:

>             // any other types ?

Not observed any yet.

**Comment by kroitor** [comment]
@npomfret thx! Will merge this shortly!

**Pull request merged**